modules/ROOT/pages/whatsnew.adoc

```diff
@@ -29,8 +29,8 @@ This can only happen during a mixed mode of SGW with a 3.2 version and an older
 
 Sync Gateway 3.2.0 now supports improved interoperability with Eventing from Couchbase Server version 7.6.3+.
 
-
-You can now create Eventing functions with read-write bindings with the source bucket associated with a Sync Gateway database.
+// Reinstate when Server 7.6.4 is available.
+// You can now create Eventing functions with read-write bindings with the source bucket associated with a Sync Gateway database.
 You can use Eventing to handle data changes that happen when applications interact and to integrate with other Couchbase services such as Data, Query and Full Text Search.
 
 For more information, see xref:server-compatibility-eventing.adoc[]
```

Comment:
Should these bits also be hidden until 7.6.4?

Reply:
It's only the read-write bindings functionality from looking at the ticket, will confirm with Priya.